crochet-macros/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
One small thing, would you mind adding the Apache-2.0 license string here?

Reply:
completely forgot about that. done